MANIFEST.in

```diff
@@ -4,3 +4,8 @@ include gimel/VERSION
 include gimel/config.json.template
 include requirements.txt
 recursive-include gimel/vendor *
+
+exclude tox.ini
+exclude requirements.in
+exclude requirements.txt
```

Comment:
In the moment exclude is the better option but later if you want to use in the setup.py it would include.
e.g http://stackoverflow.com/questions/26319101/setup-py-doesnt-see-my-requirements-txt